src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -116,7 +116,7 @@ public static String getDataflowTestApiKey() {
      * A writeable folder on the project's GCS, where java files will be staged for execution
```

Comment:
I think we should change this comment to be more generic.  
"A writable GCS path where java files can be cached and temporary test files can be written"